aries-site/src/data/structures/components.js

```diff
@@ -128,6 +128,16 @@ export const components = [
     },
     relatedContent: ['Forms', 'TextArea', 'MaskedInput'],
   },
+  {
+    name: 'ToolTip',
+    category: 'Controls',
+    description:
+      'A tooltip gives a brief, informative message when the user hovers over an element.',
+    seoDescription:
+      'A tooltip gives a brief, informative message when the user hovers over an element.',
+    sections: [],
+    relatedContent: ['Layer', 'Button'],
```

Comment:
I think Header should be related as well. It's the main use case for the Header.